flask_admin/templates/bootstrap4/admin/model/modals/create.html

```diff
@@ -32,5 +32,4 @@
 {% endblock %}
 
 {% block tail %}
-  <script src="{{ admin_static.url(filename='admin/js/bs4_modal.js', v='1.0.0') }}"></script>
```

Comment:
Great work!

Since there is nothing in the tail block, I think we should remove it.

Reply:
Done